qa/workunits/rbd/cli_generic.sh

```diff
@@ -743,6 +743,14 @@ test_migration() {
     rbd migration execute test1
     rbd migration commit test1
 
+    # testing trash
+    rbd migration prepare test1
+    rbd trash mv test1 2>&1 | grep 'cannot move migrating image to trash'
```

Comment:
Nit: I would prefer an ```expect_fail``` instead of tying the test to specific error messages (same below)